lang/clojure/Portfile

```diff
@@ -16,14 +16,14 @@ long_description    Clojure is a dynamic programming language for the JVM. \
                     dialect of Lisp, with a rich set of immutable, \
                     persistent data structures and a macro system.
 homepage            https://clojure.org
-platforms           any
+platforms           darwin
```

Comment:
`platforms darwin` is now implicit by default - you can simply delete the line instead of setting `platforms darwin`:

```suggestion
```

Reply:
thanks for the hint. I removed the 'platforms' definition

Reply:
should I squash the commits?

Reply:
Please do

Reply:
ok. I force-pushed the squashed commits...

Reply:
Why change `platforms` from `any` to `darwin` at all?

Reply:
because portlint was failing with "any"... 
https://github.com/macports/macports-base/commit/84d8683465b79829d64269ae624fe0f323006a06

Reply:
Right, that was a bug (only) in lint; the value worked correctly in all other ways. It should be restored to `any` so that we aren't redundantly building a separate archive per macOS version.

Reply:
ok. I'll make the PR...

Reply:
I can just push it directly if you'd prefer.

Reply:
the direct update will be much quicker, thanks...

Reply:
should I prepare the patch or will you do it yourself?

Reply:
Thanks. Done in d906bac.

Reply:
thanks...